workflow/scripts/featurecounts_deseq2.R

```diff
@@ -330,7 +345,8 @@ if (file.exists(ResultsFile) == FALSE) {
         dev.off()  # AVI: added to create separate pdf files
 
         ## SCATTER PLOT FOR RLOG COUNTS
-        pdf(file=snakemake@output[["plot_scatter"]],width=10,height=8)  # AVI: added to create separate pdf files
+        ScatterPlotFile <- file.path(snakemake@output[["plot_scatter"]], paste(snakemake@wildcards[["antibody"]], ".", CompPrefix, ".scatter_plots.pdf", sep=""))  # AVI: dynamically file name extensions added
```

Comment:
```suggestion
        ScatterPlotFile <- file.path(
            snakemake@output[["plot_scatter"]],
            paste(snakemake@wildcards[["antibody"]], ".", CompPrefix, ".scatter_plots.pdf", sep="")
        )  # AVI: dynamically create file names
```
Improve readability.